lib/health_checks/custom/redis_check.rb

```diff
@@ -0,0 +1,13 @@
+module HealthChecks
+  module Custom
+    class RedisCheck
+      def initialize(config)
+        @config = config
+      end
+
+      def run
+        Redis.new(@config).ping
```

Comment:
What's the timeout here? Have you checked it's 1 sec?

Reply:
must read

Reply:
There's the possibility there's no client timeout, only server one

Reply:
So how have you tested that this works?

Reply:
by scaling down the statefulSet, same for the Mongo

Reply:
http://lmgtfy.com/?q=ruby+redis
https://github.com/redis/redis-rb#timeouts

Reply:
f*ck :) somehow... Good job @yassenb :)